packages/esm-billing-app/src/billable-services/payment-history/payment-history-table.component.tsx

```diff
@@ -1,26 +1,20 @@
-import {
-  DataTableSkeleton,
-  Table,
-  TableHead,
-  TableRow,
-  TableHeader,
-  TableBody,
-  TableCell,
-  DataTable,
-} from '@carbon/react';
+import { DataTableSkeleton, Table, TableBody, TableCell, TableHead, TableHeader, TableRow } from '@carbon/react';
 import { EmptyState, ErrorState } from '@openmrs/esm-patient-common-lib';
 import React from 'react';
-import styles from './payment-history.scss';
 import { useTranslation } from 'react-i18next';
 import { useBills } from '../../billing.resource';
+import { MappedBill } from '../../types';
 import { headers } from './payment-history-viewer.component';
+import styles from './payment-history.scss';
 
 export const PaymentHistoryTable = ({
   tableData,
   paidBillsResponse,
+  renderedRows,
 }: {
   tableData: any;
   paidBillsResponse: ReturnType<typeof useBills>;
+  renderedRows: MappedBill[];
```

Comment:
```suggestion
  renderedRows: Array<MappedBill>;
```
For constistency lets have array declaration following this syntax

```ts
Array<T>
```

Reply:
Got it. From now onwards I will use the syntax.